Formula/tezos-accuser-PtKathma.rb

```diff
@@ -64,24 +64,18 @@ def install
 
       accuser="#{bin}/octez-accuser-PtKathma"
 
-      accuser_dir="$TEZOS_CLIENT_DIR"
-
-      accuser_config="$accuser_dir/config"
-      mkdir -p "$accuser_dir"
+      accuser_config="$TEZOS_CLIENT_DIR/config"
+      mkdir -p "$TEZOS_CLIENT_DIR"
 
       if [ ! -f "$accuser_config" ]; then
-          "$accuser" --base-dir "$accuser_dir" \
-                    --endpoint "$NODE_RPC_SCHEME://$NODE_RPC_ADDR" \
+          "$accuser" --endpoint "$NODE_RPC_SCHEME://$NODE_RPC_ADDR" \
```

Comment:
I'm slightly against this change because it actually hides how data directories are determined and it might be much harder to debug is something goes wrong. In general I believe that various utility scripts should be as verbose as possible

Reply:
Fair, I tried to make this a bit more clear with comments in scripts used by systemd, e.g. in `tezos-accuser-start`:
```sh
# Note: the 'TEZOS_CLIENT_DIR' env var is expected and used by the accuser
```
I didn't here because `TEZOS_CLIENT_DIR` is listed in the `EnvironmentVariables` below, do you think that it's worth adding these comments here too?